### PR TITLE
fix: Warning background in pop-up window

### PR DESCRIPTION
**Pull request by devchenyan**
issue: https://github.com/Magickbase/neuron-public-issues/issues/396

image

### CODE REVIEW

File: packages/neuron-ui/src/widgets/Dialog/dialog.module.scss [hunk 1]
```diff
@@ -50,6 +50,7 @@
   position: relative;
   max-height: calc(100vh - 260px);
   overflow-y: scroll;
+  scrollbar-width: none;
```
Comment (yanguoyu) [inline packages/neuron-ui/src/widgets/Dialog/dialog.module.scss:53]: This will affect the users who use a mouse without wheel.

**Review by Keith-CY — APPROVED** [review]

File: packages/neuron-ui/src/widgets/Dialog/dialog.module.scss [hunk 2]
```diff
@@ -49,7 +49,6 @@
   padding: 20px 16px;
   position: relative;
   max-height: calc(100vh - 260px);
```
Comment (Keith-CY) [inline packages/neuron-ui/src/widgets/Dialog/dialog.module.scss:51]: How does it look when the message is long and wrapped so the height is over the `max-height`

Comment (devchenyan) [reply packages/neuron-ui/src/widgets/Dialog/dialog.module.scss:51]: > How does it look when the message is long and wrapped so the height is over the `max-height`

image

Comment (Keith-CY) [reply packages/neuron-ui/src/widgets/Dialog/dialog.module.scss:51]: It looks good for me because we won't have such a long error message

**Comment by Danie0918** [comment]
@yanguoyu Please have a review.

File: packages/neuron-ui/src/widgets/Dialog/dialog.module.scss [hunk 3]
```diff
@@ -49,7 +49,6 @@
...
-  overflow-y: scroll;
```
Comment (yanguoyu) [inline packages/neuron-ui/src/widgets/Dialog/dialog.module.scss:52]: How about using `overflow-y: auto`? `overflow-y` 's initial value is visible

**Review by yanguoyu — APPROVED** [review]

**Comment by yanguoyu** [comment]
@silySuper For this PR, the dialog component CSS has been changed. So It's better to check whether all of the dialog styles work well.

**Review by silySuper — APPROVED** [review]

**Pull request merged**